### ISSUE TITLE
Project can't be opened nor imported anymore

### ISSUE THREAD

**Issue by katerlouis** [vuejs/vue-cli#3286]
### Version
3.3.0

### Reproduction link
[https://github.com/katerlouis/test777](https://github.com/katerlouis/test777)

### Environment info
```
(node:2574) UnhandledPromiseRejectionWarning: SyntaxError: Unexpected token } in JSON at position 214
    at JSON.parse (<anonymous>)
    at p.then.then.e (/Users/reneeschke/.nvm/versions/node/v10.7.0/lib/node_modules/@vue/cli/node_modules/envinfo/dist/envinfo.js:1:2473)
(node:2574) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:2574) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

### Steps to reproduce
Open UI and try to import the project

### What is expected?
Navigating the project as all the others

### What is actually happening?
The hover is still happening, but nothing happens on click

---
I was fiddling around with fiddling around with environment variables / custom modes etc. in a testproject– 
Sometimes building my custom library took over 180s (usually 15-20) – 

I don't know how / when exactly it "broke", but I was closing the UI sometimes when the long build happened again. The last build where the UI responded still worked, though. I could still see the UI say "Fertig" (still can't change the language of the UI, so you have cope with the german "done" :D) – the UI was still responding, hovers were still working, I could navigate to other projects, but not navigating the current project anymore. Hovering tasks still worked, but clicking them didn't do anything. 

Now I can't open it, nor import it.

I don't know if this helps in any way, but the broken project test777 has no dot behind its label, like the working project has. 
![image](https://user-images.githubusercontent.com/1983382/51047694-6f370f80-15c9-11e9-85aa-36a1a9e76967.png)

**Comment by Akryum** [comment]
As far as I can see, the project `package.json` isn't valid JSON: https://github.com/katerlouis/test777/blob/master/package.json

**Comment by LinusBorg** [comment]
yep, comments in JSON are not allowed.

**Comment by katerlouis** [comment]
Yep, that was the issue. Crazy– didn't know that comments aren't allowed in JSON. Thanks!

What exactly is happening under the hood of the GUI there? Could we catch the error and give an alert-dialogue or something? Would like to help with this :) 

I'd argue I'm not the only rookie messing up things like this. If something just stops working it adds up to the confusion this whole new world of webpack, build-steps, es6, vue, vuex, vue-router etc. etc. –

What do you think?
